frontend/public/actions/ui.ts

```diff
@@ -281,6 +283,13 @@ export const sortList = (
 
   return action(ActionType.SortList, { listId, field, func, orderBy });
 };
+export const setColumnManagementFilter = (id: string, filter: any) => {
+  const currentFilters =
+    JSON.parse(localStorage.getItem(COLUMN_MANAGEMENT_LOCAL_STORAGE_KEY)) ?? {};
```

Comment:
Need to handle parse errors.

Reply:
@spadgett ok, if we run into a parsing error retrieving the filters from the store, i'm now logging an error message to the console, and aborting the operation.